NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using NotificationApi.Common;
+using NotificationApi.Contract;
+using NotificationApi.Contract.Requests;
+using NotificationApi.Extensions;
+using NotificationApi.Services;
+using NUnit.Framework;
+
+namespace BookingQueueSubscriber.UnitTests.Mappers.NotificationMappers
```

Comment:
Perhaps name this and the folder `NotificationParameterMapper`, for consistency with the class name

Reply:
could we make it singular - `NotificationParameterMapper`

(both the folder and namespace)